runtime/src/identity.rs

```diff
@@ -793,22 +748,25 @@ decl_module! {
                                     T::AcceptTransferTarget::accept_ticker_transfer(did, auth_id),
                                 AuthorizationData::TransferTokenOwnership(_) =>
                                     T::AcceptTransferTarget::accept_token_ownership_transfer(did, auth_id),
+                                AuthorizationData::AddMultisigSigner =>
+                                    T::AddSignerMultisigTarget::accept_multi_sig_signer(Signer::from(did), auth_id),
                                 _ => Err("Unknown authorization")
                             };
                         }
                     }
                 },
-                Signer::Key(_key) => {
+                Signer::Key(key) => {
                     for auth_id in auth_ids {
                         // NB: Even if an auth is invalid (due to any reason), this batch function does NOT return an error.
                         // It will just skip that particular authorization.
                         if <Authorizations<T>>::exists((signer, auth_id)) {
-                            let _auth = Self::authorizations((signer, auth_id));
-                            // NB: Result is not handled, invalid auths are just ignored to let the batch function continue.
-                            // TODO: Add cases where signing key is authorized
-                            // let _result = match auth.authorization_data {
-                            //     _ => Err("Unknown authorization")
-                            // };
+                            let auth = Self::authorizations((signer, auth_id));
+                            //NB: Result is not handled, invalid auths are just ignored to let the batch function continue.
+                            let _result = match auth.authorization_data {
+                                AuthorizationData::AddMultisigSigner =>
+                                T::AddSignerMultisigTarget::accept_multi_sig_signer(Signer::from(key), auth_id),
```

Comment:
It looks like one more level of indentation is needed after `=>\n`.

Reply:
Thanks for catching that, I'll patch it.